### DISCUSSION TITLE
Profile duplicated if importing the .zip file

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#384] [Q&A] [answered]
If I import a single profile (.json) I'll be asked if I'd like to update this profile or create a new one. That's fine. If I import the .zip file from a previous "Export all" there is no question and my personell profile (I've just one) is duplicated. Shouldn't we ask in this case either? 

<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/a9382927-c652-49b8-9d63-d61245c4a13e"/>    <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/733c01a8-9384-4161-866c-a7ae308bc7d0"/>     <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/b37cba30-e5f4-4710-97b6-806fd8a984db"/>

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report.

Indeed we can improve the zip handling. :slightly_smiling_face:

**Comment by devemux86** [comment]
Automatic profile detection on import will be available in the next Cruiser update.

**Comment by devemux86** [comment]
@WolfgangZ42 can you please test again with the latest:
- https://github.com/devemux86/cruiser/discussions/390

**Reply by WolfgangZ42** [reply]
Works fine. Thanks! 👍

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.37](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
